File: kfcore/src/main/java/com/lq/ren/base/data/Summary.java
package com.lq.ren.base.data;

import java.util.ArrayList;
import java.util.List;

/**
 * Author lqren on 17/10/11.
 */

public class Summary {

    public String userId;

    //Daily
    public int healthTarget = -1;
    public int steps;
    public float distance;
    public float calorie;
    public int targetProgressRate;

    //fit
    public final List<Point> todayExercises = new ArrayList<>();

}

File: kfcore/src/main/java/com/lq/ren/base/data/Summary.kt
package com.lq.ren.base.data

import java.util.ArrayList

/**
 * Author lqren on 17/10/11.
 */

class Summary {

    var userId: String? = null

    //Daily
    var healthTarget = -1
    var steps: Int = 0
    var distance: Float = 0.toFloat()
    var calorie: Float = 0.toFloat()
    var targetProgressRate: Int = 0

    //fit
    val todayExercises: List<Point> = ArrayList()

}
